### PR TITLE
Lesson1 server page

### PR DESCRIPTION
**Pull request by Ansper11**
This is the lesson1 combined with our server:

- limitation: for now can only print the image for  specific pages (check the comment in the main.js line144 to see how to make a specific page to print the image) I will check for another solution later

- if any page is not work in your game or browser you can try to clean your cache

### CODE REVIEW

**Review by Nick-McGee — APPROVED** [review]

File: webserver/static/js/main.js [hunk 1]
```diff
@@ -136,17 +136,19 @@ document.getElementById("run").addEventListener('click', async function() {
     var image=""
     //check if current part have a plot 
     if (lesson[part].haveplot===true){
```
Comment (Nick-McGee) [inline webserver/static/js/main.js:138]: Are we able to have multiple plots?

**Comment by Ansper11** [comment]
We cannot print 2 plots on the same page, but if any lesson requires multiple plots we can make different pages for now. I will check if we can make pop-up windows for the graph later.

**Comment by eloiseespel** [comment]
This is coming together very nicely

**Comment by yyht6300a** [comment]
> We cannot print 2 plots on the same page, but if any lesson requires multiple plots we can make different pages for now. I will check if we can make pop-up windows for the graph later.

I don't really think we can pop up a window in the code builder.

**Pull request merged**